client/__main__.py

```diff
@@ -0,0 +1,7 @@
+import asyncio
```

Comment:
What is the advantages of using `asyncio` here?

Reply:
Are you recommend to create function and call it in this file?

Reply:
I'm just asking why should we use asyncio for client tool?

Reply:
`asyncio` needs to run async function. Mb in def `__main__`. It is redundant. I will remove `asyncio` calls in def `start_game`.

Reply:
No, I'm asking in general. Why should we use async approach instead of sync approach?

Reply:
Oh, in this side, I chose this way only to call to mind  async approach